src/components/modals/CreateTeam.js

```diff
@@ -0,0 +1,157 @@
+          onRequestClose={() => {
+            this.setModalVisible(!this.modalVisible);
+          }}
+         >
+          <CenterContainer>
+            <CreateTeamContainer>
+                    <Subtitle>Create team</Subtitle>
+
+                <StyledFlexBox>
+                    <TextBody>Team Name: </TextBody>
+                    <TextField onChangeText={teamName => this.setState({ teamName: teamName })}></TextField>
+                </StyledFlexBox>
+                <StyledFlexBox>
+                    <TextBody>Course number: </TextBody>
+                    <TextField onChangeText={courseNumber => this.setState({ courseNumber: courseNumber })}></TextField>
+                </StyledFlexBox>
+                <StyledFlexBox>
+                    <TextBody>Maximum number of participants: </TextBody>
+                    <NumberField onChangeText={maxNum => this.setState({ maxNum: maxNum })}></NumberField>
+                </StyledFlexBox>
+                <ButtonContainer>
+                    <ConfirmButton onPress={() => this.createTeam()}>
+                        <ButtonText>create</ButtonText>
+                    </ConfirmButton>
+                    <CancelButton onPress={() => this.setModalVisible(!modalVisible)}>
+                        <ButtonText>cancel</ButtonText>
+                    </CancelButton>
+                </ButtonContainer>
+            </CreateTeamContainer>
+          </CenterContainer>
+        </Modal>
+      </View>
+    );
+  }
+}
+
+//STYLED-COMPONENTS
+
+const AddButtonContainer = styled.View `
+    justify-content: center;
+    align-items: center;
+`
+
+const CenterContainer = styled.View `
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+`
+
+const TextField = styled.TextInput `
+    background-color: ${theme.COLOR_LIGHT_GRAY};
+    border-top-left-radius: 100;
+    border-bottom-left-radius: 100;
+    border-top-right-radius: 100;
+    border-bottom-right-radius: 100;
+    width: 60%;
+    padding-left: ${theme.SPACING_XSMALL};
+`
+
+const NumberField = styled(TextField) `
+    width: 30%;
+`
+
+const StyledFlexBox = styled.View `
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-vertical: ${theme.SPACING_XSMALL};
+`
+
+const ButtonContainer = styled.View `
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+`
+
+const CreateTeamContainer = styled.View `
+    flex-direction: row;
+    flex-wrap: wrap;
+    background-color: ${theme.COLOR_WHITE};
+    max-width: 85%;
+    border-radius: 12;
+    padding-vertical: ${theme.SPACING_MEDIUM};
+    padding-horizontal: ${theme.SPACING_MEDIUM};
+`
+
+const ConfirmButton = styled.TouchableOpacity `
+    background-color: ${theme.COLOR_GREEN};
+    border-top-left-radius: 100;
+    border-bottom-left-radius: 100;
+    border-top-right-radius: 100;
+    border-bottom-right-radius: 100;
+    width: 30%;
```

Comment:
can you make this 35%? the last letter for create and cancel break to newline for my smol phone

Reply:
SAD

Reply:
updated!